test/Sentry.Tests/AppDomainUnhandledExceptionIntegrationTests.cs

```diff
@@ -56,6 +59,27 @@ public void Handle_TerminatingTrue_DisposesHub()
             disposableHub.Received(1).Dispose();
         }
 
+        [Fact]
+        public void Handle_TerminatingTrue_IsHandled()
```

Comment:
This tests assert the value Handled is false when app is terminating. Would be nice to have another test that tests it's set to `null` or `true` when this is not captured through the `UnhandledExceptionhandler`.

```suggestion
        public void Handle_TerminatingTrue_IsHandledFalse()
```